### ISSUE TITLE
/Sit proc

### ISSUE THREAD

**Issue by grisker** [Wall-core/Everlook-Bugtracker#1]
When you sit you do get critted this works.
The eye of the storm talent in the elemental tree for shaman works but you cannot proc it instantly by sitting down. 
I then tested warrior talents enrage and bloodcraze, and the talents work but you cannot proc it instantly by sitting down.
Lastly I tested the paladin talent reckoning, this talent works and I was able to sit proc it . I would sit 4 times get crit 4 times, and then get 4 swings.

tldr:
Was unable to sit proc with shaman or warrior.  Paladin does work.

**Comment by Wall-core** [comment]
fixed
